app/src/main/res/xml/quran_preferences.xml

```diff
@@ -66,6 +66,15 @@
         android:persistent="true"
         android:summary="@string/prefs_night_mode_text_brightness_summary"
         android:title="@string/prefs_night_mode_text_brightness_title"/>
+
+    <com.quran.labs.androidquran.ui.preference.SeekBarBackgroundBrightnessPreference
```

Comment:
this class doesn't exist and causes error when settings is open

Reply:
#1916

Reply:
@tahmid-23 Thank you for noting this. I forgot to stage the file!
@ahmedre Thank you for the fix and improvements!